src/IRutils.cpp

```diff
@@ -365,36 +365,36 @@ std::string resultToSourceCode(const decode_results *results) {
   // Only display the value if the decode type doesn't have an A/C state.
   if (!hasACState(results->decode_type))
     output += " " + uint64ToString(results->value, 16);
```

Comment:
You missed this one. F(" ");

Reply:
Single character strings perhaps is better to be done like chars `' '`?

Reply:
Try it and see. I suspect it won't work as a 'x' is a char and a "x" is a string.
It all depends on the addition(+) method associated with Arduino's _String_ type, and c++'s `std::string` type. If string + char works in both cases, I'm more than happy to convert them to chars instead of strings.